lib/node_modules/@stdlib/blas/base/sgemv/lib/ndarray.js

```diff
@@ -72,11 +72,17 @@ function sgemv( trans, M, N, alpha, A, strideA1, strideA2, offsetA, x, strideX,
 	if ( N < 0 ) {
 		throw new RangeError( format( 'invalid argument. Fourth argument must be a nonnegative integer. Value: `%d`.', N ) );
 	}
+	if ( strideA1 === 0 ) {
```

Comment:
@ShabiShett07 In contrast to `dtrsv`, we don't have an `LDA` check for `sgemv`. So why are we adding explicit `strideA*` checks here? I don't think that these checks are needed or desired.